addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -175,6 +199,55 @@ public String getDeviceId() {
     }
 
     @Override
+    public Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException {
+
+        Command OHcommand;
+
+        if (valueSelector.getItemClass() == SwitchItem.class) {
+
+            if (valueSelector == RFXComValueSelector.COMMAND) {
+
+                switch (command) {
+                    case OFF:
+                    case GROUP_OFF:
+                        OHcommand = OnOffType.OFF;
+                        break;
+
+                    case ON:
+                    case GROUP_ON:
+                        OHcommand = OnOffType.ON;
+                        break;
+
+                    case DIM:
+                        OHcommand = IncreaseDecreaseType.DECREASE;
```

Comment:
Yes. Most core types implement State (OnOffType, OpenClosedType etc.) but a few only implement Commands - NextPreviousType, StopMoveType and IncreaseDecreaseType. This is not unreasonable and leads to a need to know whether to convert to a State or a Command.